metisp/pymetis/src/pymetis/tests/generic.py

```diff
@@ -52,3 +48,10 @@ def test_can_be_run_with_pyesorex(self, name, create_pyesorex):
         pyesorex = create_pyesorex(self._recipe)
         assert isinstance(pyesorex.recipe, cpl.ui.PyRecipe)
         assert pyesorex.recipe.name == name
+
+    @staticmethod
+    def test_pyesorex_runs_without_stderr(name, sof, create_pyesorex):
+        output = subprocess.run(['pyesorex', name, root / sof, '--log-level', 'DEBUG'],
+                                capture_output=True)
+        assert output.returncode == 0
```

Comment:
Indeed much nicer to do this check here. Now we can just run `pytest --pdb` to get here and inspect `output`